getvalue.php

```diff
@@ -1,7 +1,11 @@
 <?php
 
-header("Content-Type: application/jsonrequest");
-$file = "database.txt";
+header("Content-Type: application/json");
+$file = "database.json";
+
+if (file_exists("database.txt") && !file_exists($file)) {
+    rename("data.txt", $file);
```

Comment:
I think this is wrong

```suggestion
    rename("database.json", $file);
```